tests/copier_data/data2.yaml

```
@@ -9,6 +9,8 @@ use_windows_in_ci: true
 
 
 
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (5 > 2)

(empty-lines)